android/build.gradle

```diff
@@ -4,8 +4,8 @@ buildscript {
     ext {
         buildToolsVersion = "30.0.2"
         minSdkVersion = 21
-        compileSdkVersion = 30
-        targetSdkVersion = 30
+        compileSdkVersion = 31
```

Comment:
As per comment in telegram, we need to add some permissions: https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#declare